test/parallel/test-npm-version.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const path = require('path');
+const exec = require('child_process').exec;
+const assert = require('assert');
+
+const npmPath = path.join(
+  __dirname,
+  '..',
+  '..',
+  'deps',
+  'npm',
+  'bin',
+  'npm-cli.js'
+);
+
+exec(`${process.execPath} ${npmPath} -v`, {
+}, common.mustCall(handleExit));
+
+function handleExit(error, stdout, stderr) {
+  const code = error ? error.code : 0;
```

Comment:
Instead of checking for errors here and on the next line, can't you just `assert.ifError(error)`?